src/runtime/persistence/process_model_service.ts

```diff
@@ -18,20 +19,23 @@ const logger: Logger = Logger.createLogger('processengine:persistence:process_mo
 
 export class ProcessModelService implements IProcessModelService {
 
+  private readonly _correlationService: ICorrelationService;
```

Comment:
```suggestion
  private readonly _correlationRepository: ICorrelationRepository;
```

See my comment above.